src/PhpCodeFixer.php

```diff
@@ -141,13 +141,25 @@ public function checkDir($dir) {
      * @param Report $report
      */
     protected function checkDirInternal($dir, Report $report) {
+        if (in_array(strtolower(basename($dir)), $this->excludeList, true)) {
+            $report->addInfo(Report::INFO_MESSAGE, 'Folder ' . $dir . ' skipped');
+        }
+
         foreach (glob($dir.'/*') as $file) {
+            $path_info = pathinfo($file);
+
+            if (
+                in_array(strtolower($path_info['filename']), $this->excludeList, true)
+                || in_array(strtolower($path_info['basename']), $this->excludeList, true)
+            ) {
+                $location_type = (is_dir($file)) ? 'Folder' : 'File';
+                $report->addInfo(Report::INFO_MESSAGE, $location_type . ' ' . $file . ' skipped');

```

Comment:
I think you missed here a `continue` statement.

Reply:
Absolutely correct, the continue statement is missing.